src/runtime/interrupt/interrupt_esp32c3.go

```diff
@@ -74,7 +114,70 @@ func handleInterrupt() {
 		riscv.MSTATUS.SetBits(0x8)
 
 		// Call registered interrupt handler(s)
-		esp.HandleInterrupt(int(interruptNumber))
+		switch interruptNumber {
+		// case IRQNUM_1:
```

Comment:
@aykevl 
tinygo crash when uncommenting cases 1-5
tinygo failing in link when uncommenting case 5